mantidproject/mantid
# Fix comparisons/tests pep8 E711, E712, E713 and E714

Fixes comparisons/tests pep8 E711, E712, E713 and E714

**To test:**
- Code review, check that non of the logic has changed.
- Check that the test still pass

**_Does not need to be in the release notes.**_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
